_posts/2024-03-25-new-sponsoring-tiers.md

```diff
@@ -0,0 +1,41 @@
+---
+title: New sponsoring tiers
+summary: We refurbish the tiers to provide more to our sponsors
+author: beta-ziliani
+---
+
+We made several changes to the sponsor packages offered on the [OpenCollective sponsoring page](https://www.opencollective.com/crystal-lang).
+
+The sponsoring tiers and their perks hadn't been updated in a while and we felt it was time to freshen up.
+Particularly, we want to offer more differentiated opportunities for expressing thanks to our sponsors.
+
+The general structure of tiers hasn't changed. It's a bit more refined now with clearer naming.
+We consider three groups of sponsorship levels with different target audiences:
+
+1. **Backer:** Individuals and small organizations who love Crystal and want to support it.
+2. **Sponsor:** Individuals and organizations who want to support Crystal and maybe gain some visibility.
+3. **Corporate:** Organizations who want to support Crystal and gain support to move their business forward.
```

Comment:
If the label is singular, keep it singular.

```suggestion
1. **Backer:** An individual or small organization who loves Crystal and wants to support it.
2. **Sponsor:** An individual or organization who wants to support Crystal and maybe gain some visibility.
3. **Corporate:** An organization who wants to support Crystal and obtain support to move their business forward.
```

Reply:
Alternatively:

```suggestion
1. **Backers:** Individuals and small organizations who love Crystal and want to support it.
2. **Sponsors:** Individuals and organizations who want to support Crystal and maybe gain some visibility.
3. **Corporates:** Organizations who want to support Crystal and obtain support to move their business forward.
```

Reply:
(I also changed `gain` for `obtain`. It can also work with `get`.)

Reply:
I actually don't see any dissonance here: "The Backer tier consists of individuals and organizations." That's totally legit grammar and IMO makes the most sense.

The singular description feels awkward. The singular labels are derived from the tier names, which fits very well.